filament/backend/src/opengl/platforms/PlatformEGL.cpp

```diff
@@ -448,6 +448,11 @@ EGLConfig PlatformEGL::findSwapChainConfig(uint64_t flags, bool window, bool pbu
         configAttribs[EGL_RECORDABLE_ANDROID] = EGL_TRUE;
     }
 
+    if (flags & SWAP_CHAIN_CONFIG_MSAA_4_SAMPLES) {
+        configAttribs[EGL_SAMPLE_BUFFERS] = 1;
+        configAttribs[EGL_SAMPLES] = 4;
+    }
```

Comment:
I think if we're on a device that doesn't support 4 samples, this will fail the swapchain creation.

I think there needs to be a `isSupported" call. And we need to be very clear about what happens if MSAA_4_SAMPLES is used but not supported.

Here, it's a bit inconsistent because on on non-EGL it's ignored, but on EGL if it's not supported, it'll fail.